Cargo.toml

```diff
@@ -16,14 +34,7 @@ debug-assertions = false
 codegen-units = 16
 incremental = true
 
+# The profile that 'cargo dist' will build with
 [profile.dist]
 inherits = "release"
-
-[workspace.metadata.dist]
-# The preferred Rust toolchain to use in CI (rustup toolchain syntax)
-# Here, we use the MSRV.
-rust-toolchain-version = "1.57.1"
-# CI backends to support (see 'cargo dist generate-ci')
-ci = ["github"]
-# Target platforms to build apps for (Rust target-triple syntax)
-targets = ["x86_64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-pc-windows-msvc"]
\ No newline at end of file
+lto = "thin"
```

Comment:
I'm not against changing our LTO config, but I think we should move this to our default `profile.release` for consistency.

Reply:
That would make sense yes! This was just the default generated by `cargo dist init`. Do you want me to move it to `profile.release` in this PR?